src/twisted/python/lockfile.py

```diff
@@ -50,7 +50,7 @@ def unique():
         ERROR_ACCESS_DENIED = 5
         ERROR_INVALID_PARAMETER = 87
 
-        def kill(pid, signal):
+        def kill(pid, signal):  # type: ignore[misc]
```

Comment:
why should this be ignored? I think that will kill always have `ini, int`

Reply:
@adiroiban I explained this a bit in the commit log for https://github.com/twisted/twisted/pull/1297/commits/c6f85c065ef91ccdd5848cab4e30c67b36cfcbe4

```
Tell mypy to ignore functions which try to replace functions implemented

in C.

For example, 'os.kill' is implemented in C,
and as far as mypy is concerned, has a different signature than
def kill(pid, signal)
```

If you do:

```
from os import kill
help(kill)
```

you will see that the signature of the function is:

```
kill(pid, signal, /)
```

The only way to implement a `/` parameter outside of C, is to use Python 3.8 or higher which implements PEP 570.  So in this case it was easier to put the comment for mypy to ignore this line.

Reply:
Thanks for the explanation. TIL :)

I think that this information should not be put in a commit message, or a review response, but rather as a code comment.

At the same time, I think that mypy should have some special annotation for positional only arguments, and to not just use an ignore annotation.
I see that mypy only got this for python3.8 https://github.com/python/mypy/pull/6900 without any backward compatibility.

So I think that this needs a comment, at least to indicate that this can be removed once Twisted drops support for 3.5, 3.6 and 3.7.

Thanks